src/store/modules/geometry/actions/createFaceFromPoints.js

```diff
@@ -55,17 +55,16 @@ export default function createFaceFromPoints(context, payload) {
   }
 
   withPreservedComponents(context, currentStoryGeometry.id, () => {
-
     newGeoms.forEach(newGeom => context.dispatch('replaceFacePoints', newGeom));
 
     // save the face and its descendent geometry
     storeFace(faceGeometry, target, context, existingFace);
 
+    context.dispatch('trimGeometry', { geometry_id: currentStoryGeometry.id });
```

Comment:
A comment explaining the importance of the ordering would be nice for the future

Reply:
Agreed. I'll add something